banners/desktop/banner_ctrl.ts

```diff
@@ -45,6 +46,8 @@ const app = createVueApp( BannerConductor, {
 	},
 	resizeHandler: new WindowResizeHandler(),
 	banner: Banner,
+	fallbackBanner: FallbackBanner,
+	minWidthForMainBanner: 800,
```

Comment:
This could probably go into the banner config. Actually we probably should be initialisiing that early in the entry point then using values from it so there's no repetition and the values ae more descriptive when passed into the various constructors and props.

Reply:
Sorry, I don't understand this comment - which "banner config" do you mean? `campaign_info.toml`? I'm not sure if we should put too much in there, I think the entry point is a better place to put such "configuration".

Reply:
No, there's a bannerConfig prop. It also made me think that we could be defining that separately in the entry points eg:

```js
const bannerConfig = {
    minAllowedWidth: 400,
    minWidthForMainBanner: 800,
    bannerDelay: 7500,
    transitionDuration: 1000
};
```

Reply:
That makes perfect sense and is a good idea, thanks!